shared/templates/static/oval/accounts_umask_etc_bashrc.xml

```diff
@@ -1,7 +1,4 @@
 <def-group>
```

Comment:
It's been awhile since I visited the templating system. Aren't these files dynamically generated now? Should they be in the repo at all? And if they should, would it be appropriate to update the comments to edit shared/templates/template_OVAL_umask?